packages/salesforcedx-vscode-replay-debugger/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "lib": ["dom", "es2016"],
```

Comment:
Do you need to use dom? I'm not sure if you need it since you are not using anything from the web UI.

Reply:
I can remove; it's in all other packages.